Source/WebCore/page/TextAnimationTypes.h

```diff
@@ -37,6 +43,7 @@ enum class TextAnimationType : uint8_t {
 
 struct TextAnimationData {
     TextAnimationType style;
+    TextAnimationRunMode runAnimation;
```

Comment:
```suggestion
    TextAnimationRunMode runMode;
```

Better matches the type.